pkg/target/target_test.go

```diff
@@ -430,3 +438,144 @@ func TestProcessData(t *testing.T) {
 		})
 	}
 }
+
+var fooMatch = match.Match{
```

Comment:
Use a function to return these. As-is the tests are sharing mutable variables. If one test mutates them, we'll get nondeterministic test failures.

Reply:
done.